sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -44,7 +44,8 @@ public void resetParams() {
     
     public void handleInitResult(boolean val) {
     	setInitSuccess(val);
-    	session.onSecurityInitialized();
+		if (session != null)
+			session.onSecurityInitialized();

```

Comment:
If it's possible this gets called without a session attached to it, do we need to fire off the `session.onSecurityInitialized();` callback when a session is set? [See where we handle setting a session](https://github.com/smartdevicelink/sdl_android/blob/41a6698a2cbd7b3428fc3be55efe30fc21c3965a/sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java#L55)